lib/champaign_queue/clients/direct.rb

```diff
@@ -37,7 +37,7 @@ def uri
       end
 
       def ak_processor_url
-        ENV['AK_PROCESSOR_URL']
+        Settings.ak_processor_url
```

Comment:
Not in this file. There were actually two settings which I'd accidentally combined into one. This should stay as the AK processor URL (which is the URL of the app that processes messages off the queue, like you'd expect it to be).